nmigen_boards/arrow_deca.py

```diff
@@ -0,0 +1,86 @@
+import os
+import subprocess
+
+from nmigen.build import *
+from nmigen.vendor.intel import *
+from .resources import *
+
+
+__all__ = ["ArrowDECAPlatform"]
+
+
+class ArrowDECAPlatform(IntelPlatform):
+    device      = "10M50DAF484" # MAX 10
+    package     = ""
```

Comment:
Seems like `F484` is the package? Or does that not work for some reason?

Reply:
No, that should work. I just wasn't entirely sure how the SKU should be split up, but I can fix that and create another pull request if there is nothing else to be addressed.